code/modules/power/singularity/singularity.dm

```diff
@@ -470,17 +474,16 @@
 		if(M.flags & INVULNERABLE)
 			continue
 		if(M.stat == CONSCIOUS)
-			if (istype(M,/mob/living/carbon/human))
+			if(istype(M,/mob/living/carbon/human))
 				var/mob/living/carbon/human/H = M
 				if(istype(H.glasses,/obj/item/clothing/glasses/meson) && current_size != 11)
-					H << "<span class=\"notice\">You look directly into The [src.name], good thing you had your protective eyewear on!</span>"
+					H << "<span class='notice'>You stare directly into \the [src], good thing you had your protective eyewear on!</span>"
 					return
 				else
-					H << "<span class=\"warning\">You look directly into The [src.name], but your eyewear does absolutely nothing to protect you from it!</span>"
-		M << "<span class='danger'>You look directly into The [src.name] and feel [current_size == 11 ? "helpless" : "weak"].</span>"
+					H << "<span class='warning'>You stare directly into \the [src] but your eyewear does absolutely nothing to protect you from it!</span>"
+		M.visible_message("<span class='danger'>[M] stares blankly at \the [src]!</span>", \

```

Comment:
Indent this under the if and make sure it doesn't happen if you get the meson message.